app/src/main/java/org/thoughtcrime/securesms/mms/LegacyMmsConnection.java

```diff
@@ -186,22 +189,22 @@ protected CloseableHttpClient constructHttpClient() throws IOException {
   }
 
   protected byte[] execute(HttpUriRequest request) throws IOException {
-    Log.w(TAG, "connecting to " + apn.getMmsc());
+    Log.i(TAG, "connecting to " + apn.getMmsc());
 
     CloseableHttpClient   client   = null;
     CloseableHttpResponse response = null;
     try {
       client   = constructHttpClient();
       response = client.execute(request);
 
-      Log.w(TAG, "* response code: " + response.getStatusLine());
+      Log.i(TAG, "* response code: " + response.getStatusLine());
 
       if (response.getStatusLine().getStatusCode() == 200) {
         return parseResponse(response.getEntity().getContent());
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the resource returned by the following method call: *getContent*. Execution paths that do not contain closure statements were detected. Ensure the object returned by *getContent()* is closed in a try-finally block or declared in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).